lib/senders/templates/verification_reminder_second.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+  <title>{{t "Firefox Accounts"}}</title>
```

Comment:
Change to:
> Final Reminder: Confirm your email to activate your Firefox Account

Reply:
(and it was "Still there?" for this one)

Reply:
Thanks for clarifying. Let's stick to my suggestion. It expresses urgency and has a clear call to action.